.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,20 @@
+name: reviewdog
+on: [pull_request]
```

Comment:
Since we are doing a "lint" workflow on master (or dev, I can't remember), then shouldn't we add those too (i.e the ones already added in the other workflow) ?

Reply:
Having two different workflows allows them to run in parallel, it's faster this way. It could easily be added as a job to the lint workflow though, at the cost of waiting longer on pull requests automatic tests.